### PR TITLE
Fix b'name' notation used in script output

### LINKED ISSUE #4: b'name' notation used in script output
I just upgraded from 13 to 14, and now all the output uses the b'name' notation for file and directory names.  This is for both --analyze, and --migrate --pretend.  I'm no Python user, but a bit of searching suggests that if you use b'name' in the script, you need to do something before outputting it, or else the literal b'' will be included.  Is there any chance this is just due to some wayward configuration of my setup?

### PR DESCRIPTION
**Pull request by Jamim**
Hello @mgorny,

This PR fixes #4.
Hope it might help.

Cheers!

### CODE REVIEW

File: unsymlink-lib [hunk 1]
```diff
@@ -438,41 +463,41 @@ class MigrationState(object):
...
+                log('Removing: {} ...', lib_new)
                 p = subprocess.Popen([b'rm', b'-rf', b'--', lib_new])
                 if p.wait() != 0:
                     rm_failed = True
 
         if rm_failed:
-            print('', file=sys.stderr)
-            print('An error occurred while cleaning up the "lib.new" directories.', file=sys.stderr)
-            print('This message should be preceded by error messages from the "rm"', file=sys.stderr)
-            print('utility.')
```
Comment (Jamim) [inline unsymlink-lib:472]: `, file=sys.stderr` was missed here.

File: unsymlink-lib [hunk 2]
```diff
@@ -552,22 +577,22 @@ class MigrationState(object):
...
-                    print('rm -d {}'.format(os.path.join(lib64, p)))
+                    log_to_stdout('rm -d {}', os.path.join(lib64, p))
                 elif not nonfatal_remove(os.path.join(lib64, p)):
                     rm_failed = True
 
         if rm_failed:
-            print('', file=sys.stderr)
-            print('An error occurred while cleaning up the "lib64" directories.', file=sys.stderr)
-            print('This message should be preceded by more specific error messages', file=sys.stderr)
-            print('(interspersed with progress output).')
```
Comment (Jamim) [inline unsymlink-lib:567]: `, file=sys.stderr` was missed as well.

File: unsymlink-lib [hunk 3]
```diff
@@ -11,9 +12,33 @@ import subprocess
 import sys
 
 
+def decode(value):
+    if isinstance(value, bytes):
+        return value.decode('utf-8', errors='replace')
```
Comment (mgorny) [inline unsymlink-lib:17]: I think `backslashreplace` will be better here.

**Review by mgorny — APPROVED** [review]
Thanks. This actually saves me from going through the whole code ;-). I'll do a few cosmetic changes and merge it.

File: unsymlink-lib (hunk 3 above)
Comment (Jamim) [reply unsymlink-lib:17]: Unfortunately, it causes `UnicodeDecodeError` for Python 2.7.
Please see #7.

**Pull request closed without merging**